### DISCUSSION TITLE
how does rich text work? field type? binding?

### DISCUSSION THREAD

**Discussion by dncpax** [Budibase/budibase#2952] [Help]
Hi there.
I'm trying to have a multi-line text area, and thought of using a rich text input, and binding to a datasource text field.
This gives me an error of wrong data type.
So, my question is really how to use rich text input bindings?

Thanks.

**Comment by mjashanks** [comment]
You know - I'll have to look into this. I seem to remember that this was once a Rich Text field, with formatting options. But that seems to have gone...

**Reply by mjashanks** [reply]
logged an issue: https://github.com/Budibase/budibase/issues/2967

I don't think we will have a rich text field soon - but we should at least not pretend that it's there now, as that's confusing :)

**Reply by dncpax** [reply]
ah, ok. happens to the best ;)
Thing is I only wanted a multi-line so the existing "rich text" would work fine, but I can't bind it to any field. The dropdown for fields is empty... typing a field does not work either...

**Reply by mjashanks** [reply]
Sorry, I didn't answer your question!

I see what you mean. The Rich Text only binds to columns of type "Long Form Text" - I think this is a hangover from the days when the Rich Text actually displayed Rich Text. I'll get this logged as well.
